src/components/molecule_dict_link_button_chunk/index.tsx

```diff
@@ -20,6 +21,7 @@ const DictLinkButtonChunk: FC<Props> = ({ wordId }) => {
       <JaKoNaverDictionaryLinkButton wordId={wordId} />
       <EnDictionaryDotComDictLinkButton wordId={wordId} />
```

Comment:
Can you place here?
between `<JaKoNaverDictionaryLinkButton wordId={wordId} />` and `<EnDictionaryDotComDictLinkButton wordId={wordId} />`